vrecord

```diff
@@ -572,6 +572,110 @@ _writeingestlog(){
     fi
 }
 
+# create a png of qc data graphs for quick assessment
+_qcgraphimage(){
+    # get audio data
+    qctools_CONFIGFILE_A="$(/usr/bin/mktemp /tmp/astats_A_XXXXXXXXX).csv"
+    gzcat "$1" | perl -nle 'print if not m{lavfi.(?!astats.Overall.Min_level|astats.Overall.Max_level|astats.Overall.Peak_level|aphasemeter.phase)}' | xml select -t -m "//ffprobe:ffprobe/frames/frame[@media_type='audio']" \
+       -v "@pkt_pts_time" -o " " \
+       -v "tag[@key='lavfi.astats.Overall.Max_level']/@value" -o " " \
+       -v "tag[@key='lavfi.astats.Overall.Min_level']/@value" -o " " \
+       -v "tag[@key='lavfi.astats.Overall.Peak_level']/@value" -o " " \
+       -v "tag[@key='lavfi.aphasemeter.phase']/@value" -n > "${qctools_CONFIGFILE_A}"
+    # get video data
+    qctools_CONFIGFILE_V="$(/usr/bin/mktemp /tmp/signalstats_V_XXXXXXXXX).csv"
+    gzcat "$1" | perl -nle 'print if not m{lavfi.(?!signalstats.BRNG|signalstats.TOUT|signalstats.SATMAX|signalstats.SATHIGH|signalstats.SATLOW|signalstats.SATMIN|ssim.Y|ssim.U|ssim.V)}' | xml select -t -m "//ffprobe:ffprobe/frames/frame[@media_type='video']" \
+       -v "@pkt_pts_time" -o " " \
+       -v "tag[@key='lavfi.signalstats.TOUT']/@value" -o " " \
+       -v "tag[@key='lavfi.signalstats.BRNG']/@value" -o " " \
+       -v "tag[@key='lavfi.signalstats.SATMAX']/@value" -o " " \
+       -v "tag[@key='lavfi.signalstats.SATHIGH']/@value" -o " " \
```

Comment:
SATHIGH and a few other SAT values aren't used below so they could be removed here and from the perl parsing above to save some time